swap/swap.go

```diff
@@ -491,6 +491,26 @@ func (s *Swap) ReceivedCheques() (map[enode.ID]*Cheque, error) {
 	return cheques, nil
 }
 
+// Liquidbalance returns the liquidBalance of the chequebook of the peer. If no peer is given, it will return the balance of the node itself
+func (s *Swap) Liquidbalance(peer enode.ID) (int64, error) {
+	if peer == (enode.ID{}) {
+		balance, err := s.contract.LiquidBalance(nil)
+		if err != nil {
+			return 0, err
+		}
+		return balance.Int64(), nil
+	}
+	var otherSwap contract.Contract
+	if swapPeer := s.getPeer(peer); swapPeer != nil {
+		otherSwap, err := contract.InstanceAt(swapPeer.contractAddress, s.backend)
```

Comment:
This `err` should probably be checked or explicitly ignored if you don't care about it.

Reply:
Of course. Thanks for the catch!